canonical-web-and-design/charmhub.io
# Modify p-card to use the p-button

## Done

- Add p-button to cards
- Override some of the default card behaviour on hover and click

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit http://0.0.0.0:8045/store
- Hover over cards.
- Tab through the page
- Click on a card
- Things should work as expected


## Issue / Card

Fixes #67
